tests/integration/test_integration_all_rust.py

```diff
@@ -634,12 +634,10 @@ async def fixture_registered_test_client(ws_config) -> AsyncGenerator:
     log.debug("🐍 Test Client Disconnected")
 
 
+@pytest.mark.sentry
```

Comment:
Since the team was unfamiliar with this feature of pytest, let's add a quick document reference so that others can also benefit. We only really need to add this once. We could add it before the first test or to the `pyproject.toml` file. 

(e.g. 
```suggestion
# Selectively include or exclude this test based on `mark` See
# https://docs.pytest.org/en/7.1.x/example/markers.html
@pytest.mark.sentry
```